[WIP] Nodejs Bot Guide (#1998)
discordeno/discordeno: site/docs/nodejs/initial-setup.md

---
sidebar_position: 2
---
# Initial Setup
## Config File
On long-term you should save your configs in a `.env` file. Out of simplicity we are saving it on a `config.json` file.

Create a file named `config.json` in your project folde and insert the following content:
```json
{
    "token": "YOUR_TOKEN_HERE",
    "prefix": "!"
}
```

## Create Main File
Create a file named `index.js` in your project folder and insert the following content:
```js
const Discord = require('discordeno');
const config = require('./config.json')

const client = Discord.createBot({
    events: {
        ready() {
            console.log("Successfully connected to gateway");
        },
        messageCreate(client, message) {
            if(message.content === "!ping") {
                client.helpers.sendMessage(message.channelId ,{content: "pong"});
            }

>             console.log(`Recieved message: ${message.content || message.embeds}`);

Author: removed

---
sidebar_position: 4
---

# Initial Setup

## Config File

> On long-term you should save your configs in a `.env` file. Out of simplicity we are saving it in a `config.json` file.

Reviewer: ```suggestion
Ideally, you should save your configs in a `.env` file. Out of simplicity for this guide, we are saving it on a `config.json` file.
```
